src/System.Management.Automation/engine/CommandCompletion/CompletionHelpers.cs

```diff
@@ -55,36 +55,68 @@ internal static IEnumerable<CompletionResult> GetMatchingResults(
             }
         }
 
+        /// <summary>
+        /// Handles and processes a string that starts and/or ends with single or double quotes.
+        /// Removes the front quote if present, and optionally the back quote if it matches the front quote.
+        /// Updates the input string by reference and returns the type of quote used.
+        /// </summary>
```

Comment:
I think details should be in remark section.
Here we need simple description like: remove wrapping quotes from string and return the quote if it is present.

Reply:
Fixed in d0c847f40892f642219108fa7a8198270c9d52c9